# first --> 

# name=input('Enter Your Name : ')

# print('Good Afternoon ,'+name)

# second 

# name= input('Enter Your Name : ')
# date= input('Enter Your Name : ')

# letter = '''\n \n Dear <!name!> \n \t you are selected \n \t <!date!> '''

# letter=letter.replace('<!name!>',name)

# letter=letter.replace('<!date!>',date)

# print(letter)


# # Third 

# st="This is string with 2 space    "

# ds=st.find('  ')

# print(ds)

# # Third 

# st="This is string with 2  space "

# ds=st.replace('  ',' ')

# print(ds)


# latter="this is a good news ! Thanks !"

# formates="this is a good news ! \n \t Thanks !"
# print(formates)